web3/encoding.nim

```diff
@@ -11,6 +11,15 @@ import
   std/macros,
   stint, ./eth_api_types, stew/[assign2, byteutils]
 
+func encode*(x: EthereumUint32): seq[byte] =
+  let numTargetBytes = 32 div 8
+  let paddingBytes = 32 - numTargetBytes
+    ## the Ethereum ABI imposes a 32 byte width for every type
+  let paddingZeros = newSeq[byte](paddingBytes)
+  let ret = paddingZeros & @(x.toByteArrayBE())
+  let retHex = ret.toHex()
```

Comment:
Why compute `retHex` if it's not used?

Reply:
https://github.com/status-im/nim-web3/pull/186/commits/9de5a8ceec0241d074466f1e36dcd20250f5a0b2